### PR TITLE
Adding edit in github link to menubar

### PR DESCRIPTION
**Pull request by hardvain**
Thanks to @budziq for pointers.

### CODE REVIEW

File: theme/index.hbs [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        </div>
+
+
+        <!-- Local fallback for Font Awesome -->
+        <script>
+            if ($(".fa").css("font-family") !== "FontAwesome") {
+                $('<link rel="stylesheet" type="text/css" href="_FontAwesome/css/font-awesome.css">').prependTo('head');
+            }
+
+            // START - Rust Cookbook customization
```
Comment (steveklabnik) [inline theme/index.hbs]: This should probably be RBE, not "Rust Cookbook" :)

**Comment by hardvain** [comment]
This was copied from Rust cookbook. The index.html from master of mdbook actually changes the colour of the print icon on the right corner to blue. Not sure if this is intended. So copied from rust-cookbook repo. Will modify the PR to update the comments. Do you want it to be updated with the latest index.hbs from mdbook?

**Comment by budziq** [comment]
> This was copied from Rust cookbook. 

Cookbook's index.hbs should be in sync with the latest mdbook one but I would still copy It from upstream there to avoid copying any "cookbookisms" :)

**Comment by steveklabnik** [comment]
@hardvain 

> Do you want it to be updated with the latest index.hbs from mdbook?

Yup, that'd be great! As @budziq says it's *probably* the latest, but let's make sure 😄

**Comment by hardvain** [comment]
@steveklabnik Updated to the latest index.hbs from mdbook repo.

**Pull request merged**